VoodooSMBus/Info.plist

```diff
@@ -37,6 +37,15 @@
 		</dict>
 		<key>ELANTouchpadDriver</key>
 		<dict>
+			<key>Preferences</key>
+			<dict>
+				<key>afterTypingCooldownMs</key>
+				<integer>500</integer>
+				<key>ignoreVoodooDisableTrackpad</key>
+				<false/>
+				<key>palmDetection</key>
```

Comment:
Could you rename this to `disableWhileTyping` ? Palm detection would be actually detecting if its a palm based on the touch ellipses and I want to add that later.